cli/popper/utils.py

```diff
@@ -83,25 +84,9 @@ def get_items(dict_object):
         yield key, dict_object[key]
 
 
-def get_project_root():
-    """Tries to find the root of the project with the following heuristic:
-
-      - Find the .git folder in cwd
-
-    Returns:
-        project_root (str): The fully qualified path to the root of project.
-    """
-    base, _ = exec_cmd('git rev-parse --show-toplevel', ignore_error=True)
-
-    if not base:
-        fail("Unable to find root of project. Initialize repository first.")
-
-    return base
-
-
 def write_config(config):
     """Writes config to .popper.yml file."""
-    config_filename = os.path.join(get_project_root(), '.popper.yml')
+    config_filename = os.path.join(scm.get_root_folder(), '.popper.yml')
```

Comment:
We can also take the folder where the config is to be written as an argument, so that we don't need to import scm

Reply:
@JayjeetAtGithub we can do this (see above) so that we don't need to import `scm` in `utils.py`. In this way, we don't need to import `utils` from within the `exception` block.

Reply:
@ivotron But then how do we get root project directory in utils.py ?

Reply:
whenever the `write_config()` and `is_popperized()` functions are called. So these two functions need to receive the folder now as an argument. In this way we don't need to make import `scm` from within the `utils.py` file

Reply:
@ivotron Sorry for that , i understood it . Please review